packages/engine/Source/Scene/DynamicAtmosphereLightingType.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Atmosphere lighting effects (sky atmosphere, ground atmosphere, fog) can be
+ * further modified with dynamic lighting from the sun or other light source
+ * that changes over time. This enum determines which light source to use.
+ *
+ * @enum {number}
+ */
+const DynamicAtmosphereLightingType = {
+  /**
+   * Do not use dynamic atmosphere lighting. Atmosphere lighting effects will
+   * be lit from directly above rather than using the scene's light source.
+   *
+   * @type {number}
+   * @constant
+   */
+  OFF: 0,
```

Comment:
This is a bit picky, but other enums opt for `NONE` for the negative case, ie. `HeightReference.NONE`.